application/config.py

```diff
@@ -20,7 +20,7 @@
 class Config:
     DEBUG = False
     LOG_LEVEL = logging.INFO
-    ENVIRONMENT = os.environ.get("ENVIRONMENT", "PROD")
+    ENVIRONMENT = os.environ.get("ENVIRONMENT", "PRODUCTION")
```

Comment:
Would it be safer to use `os.environ["ENVIRONMENT"]` rather than having any default, just to force us to always explicitly define this environment variable?